packages/fx-core/src/plugins/resource/simpleauth/utils/common.ts

```diff
@@ -107,13 +107,27 @@ export class Utils {
       Constants.AadAppPlugin.id,
       Constants.AadAppPlugin.configKeys.teamsWebAppId
     ) as string;
-    const endpoint = Utils.getConfigValueWithValidation(
-      ctx,
-      isLocalDebug ? Constants.LocalDebugPlugin.id : Constants.FrontendPlugin.id,
-      isLocalDebug
-        ? Constants.LocalDebugPlugin.configKeys.endpoint
-        : Constants.FrontendPlugin.configKeys.endpoint
-    ) as string;
+
+    let endpoint: string;
+    if (!isArmSupportEnabled()) {
+      endpoint = Utils.getConfigValueWithValidation(
+        ctx,
+        isLocalDebug ? Constants.LocalDebugPlugin.id : Constants.FrontendPlugin.id,
+        isLocalDebug
+          ? Constants.LocalDebugPlugin.configKeys.endpoint
+          : Constants.FrontendPlugin.configKeys.endpoint
+      ) as string;
+    } else {
+      if (isLocalDebug) {
+        endpoint = Utils.getConfigValueWithValidation(
+          ctx,
+          Constants.LocalDebugPlugin.id,
+          Constants.LocalDebugPlugin.configKeys.endpoint
+        ) as string;
+      } else {
+        endpoint = getArmOutput(ctx, Constants.ArmOutput.frontendEndpoint) as string;
```

Comment:
ARM deployment already set these settings, so no need to config the web app any more. We can keep this logic for now, and update related logic in next week.